### DISCUSSION TITLE
'Alt Gr' key not working

### DISCUSSION THREAD

**Discussion by TomSchammo** [rhinstaller/anaconda#3843] [Q&A]
I recently installed fedora 35 and I don't use a US keyboard layout. I use a swiss-german keyboard layout and the `@` symbol can be accessed by pressing `alt gr` + `2`.

It seems that the `alt gr` modifier does not work in the installer, it din't when I tried to type `@`, but I also tried to use other combinations and they all did not yield the intended result.

**Comment by jkonecny12** [comment]
Hi @TomSchammo , could you please tell me what ISO are you using for the installation? Is that Live media or net install boot.iso?

**Reply by TomSchammo** [reply]
I downloaded the "Fedora 35: x86_64 DVD ISO" from https://getfedora.org/en/workstation/download/ and put in on a USB stick.

**Reply by jkonecny12** [reply]
In that case it's kind of expected. Anaconda keyboard layout switching doesn't work well on Wayland. See more info here:
https://bugzilla.redhat.com/show_bug.cgi?id=2016613

We are working on resolving this issue. Sorry for the inconvenience.

**Reply by TomSchammo** [reply]
I feel like I also tested mod keys when changin the layout in the testing area and they worked. But I'm probably mistaken then...
Thanks for clarifying!
